### ISSUE TITLE
Old version object edition error

### ISSUE THREAD

**Issue by ANSSI-BSOD** [MISP/MISP#4238]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Feature Request
| OS version (server)       | X 
| OS version (client)       | X
| PHP version               | X
| MISP version / git hash   | 2.4.101
| Browser                   | X

### Expected behavior
A process to inject old definition of MISP objects into the server/database.
AND/OR
MISP instance already have all definitions of MISP objects.

### Actual behavior
User, from the api, will be able to add an object with a version which is not in the server.

For example with a MISP instance (installation from scratch) with file object version 9 only.
From API, users will be able to push file object version 7 but this object won't be editable because the server says "invalid template".
Old objects will be importable into MISP but users won't be able to edit them. 

### Steps to reproduce the behavior
- Install the last MISP instance
- Install pymisp
- Downgrade version of one misp object
- Push this object with pymisp
- Try to edit it from GUI

=> Error

**Comment by iglocska** [comment]
Indeed, generally we keep all of the old versions of the template but there's no way to add them after the fact. Makes sense.
